app/src/main/java/com/nervousfish/nervousfish/activities/ChangeContactActivity.java

```diff
@@ -27,7 +27,7 @@
 import cn.pedant.SweetAlert.SweetAlertDialog;
 
 /**
- * The ContactActivity shows the contacts information and his public keys.
+ * This activity allows the user to change attributes of contacts
```

Comment:
Class JavaDoc should make sense without the class itself, so could you start with "{@link Activity} that..."